experimental/logging/log_listener.go

```diff
@@ -68,7 +82,7 @@ type flusher interface {
 // experimental.FunctionListener.
 func (f *loggingListenerFactory) NewListener(fnd api.FunctionDefinition) experimental.FunctionListener {
 	exported := len(fnd.ExportNames()) > 0
-	if (f.hostOnly || f.fsOnly) && // choose functions defined or callable by the host
+	if (f.hostOnly || f.scopes != LogScopeNone) && // choose functions defined or callable by the host
```

Comment:
LogScopeNone comparisons in this version of the change are wrong. We need as @evacchi suggested offline an all bits set LogScopeAny and do it the other way.